node/src/Pos/Wallet/Web/Pending/Updates.hs

```diff
@@ -53,3 +54,13 @@ resetFailedPtx curSlot ptx@PendingTx{..}
               , _ptxSubmitTiming = mkPtxSubmitTiming curSlot
               }
     | otherwise = ptx
+
+-- | If given pending transaction is not yet confirmed, cancels it.
+cancelApplyingPtx :: HasConfiguration => PendingTx -> PendingTx
+cancelApplyingPtx ptx@PendingTx{..}
+    | PtxApplying poolInfo <- _ptxCond =
+          ptx { _ptxCond = PtxWontApply reason poolInfo
+              }
+    | otherwise = ptx
+  where
+    reason = "Canceled manually"
```

Comment:
Yep, good.

Reply:
By the way, reason is not displayed in response of the relevant API call.
Now, when there are more than one possible reason for cancellation, maybe `reason` is worth adding (possibly breaking compatibility with exchanges)?